dockers/docker-platform-monitor/Dockerfile.j2

```diff
@@ -48,6 +49,9 @@ RUN pip2 install enum34
 RUN pip2 install thrift==0.13.0
 RUN pip3 install thrift==0.13.0
 
+# We install the libpci module in order to be able to do PCI transactions
+RUN pip3 install libpci
```

Comment:
Is only the Python 3 version of `libpci` module needed, or does the Python 2 version need to be installed as well?

Reply:
Hi Sai, this is technically a cherry-pick of the original PR which only contained Python3 version of libpci, so I thought I would keep it consistent and submit a new PR in the event that Python2 support is needed. Thoughts?

Reply:
I think considering you are already manually making a PR, it's better to add it in now if it's needed (so that it'll go in as a single commit) rather than splitting it into two separate commits.